src/lib/apiErrorHandler.js

```diff
@@ -36,7 +36,12 @@ export const handleError = async (req, res, error, logContext) => {
   }
   if (error instanceof Sequelize.Error) {
     await handleSequelizeError(req, res, error, logContext);
+  } else if (error?.stack) {
+    // Log with call stack.
+    logger.error(`${logContext.namespace} - UNEXPECTED ERROR - Call Stack: ${error.stack}`);
```

Comment:
Can we also log the whole error object? (fine to log the stack separately)

Just in case

e.g. 
```js

logger.error(`${logContext.namespace} - UNEXPECTED ERROR - ${error}`);
logger.error(`Call Stack: ${error.stack}`);
```

Reply:
Actually we don't need to the error message is at the top of the call stack.